kayak_ui_macros/src/widget.rs

```diff
@@ -71,11 +71,12 @@ impl Widget {
                     let widget_block =
                         build_widget_stream(quote! { built_widget }, constructor, 0, false);
                     (
-                        entity_id,
+                        entity_id.clone(),
```

Comment:
`entity_id` is copy so we don't need to clone here.

Reply:
`entity_id` here is a `TokenStream` which does is not copy

```
error[E0382]: borrow of moved value: `entity_id`
  --> kayak_ui_macros\src\widget.rs:75:25
   |
68 |                   let (entity_id, props, constructor) =
   |                        --------- move occurs because `entity_id` has type `TokenStream2`, which does not implement the `Copy` trait       
...
74 |                           entity_id,
   |                           --------- value moved here
75 | /                         quote! {{
80 | |                         }},
   | |__________________________^ value borrowed here after move
   |
```